ldi-orchestrator/ldio-connectors/ldio-http-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioHttpOutAutoConfig.java

```diff
@@ -28,9 +29,10 @@ public LdiComponent configure(ComponentProperties config) {
 			final RequestExecutor requestExecutor = new LdioRequestExecutorSupplier().getRequestExecutor(config);
 
 			String targetURL = config.getProperty("endpoint");
+			String contentType = config.getOptionalProperty("content-type").orElse("text/turtle");
+			var properties = new LdiRdfWriterProperties(config.extractNestedProperties(RDF_WRITER).getConfig()).withLang(RDFLanguages.nameToLang(contentType));
```

Comment:
Why is this needed? Isn't this done by the "setLangFromProperties()" in the constructor of "LdiRdfWriterProperties". If you are thinking, this logic shouldn't be in the constructor, I agree. But it is..